README.md

```diff
@@ -4,7 +4,13 @@ See the instructions to run on (Windows)[./WINDOWS.md].
 
 ## Install
 
-`npm install`
```

Comment:
Any way to make `npm install` fail?

Reply:
I don't think there is

`lavamoat/allow-scripts` blocks `preinstall` and `postinstall` scripts, so we can't make it fail

Reply:
Great! Thanks!